content/tutorials/r_olre/index.Rmd

```diff
@@ -0,0 +1,98 @@
+---
+title: "Observation level random effects"
+description: ""
```

Comment:
Nice work @hansvancalster to move the Rpubs tutorials here! Can you put something short in the description field (in all cases)? It shows up on the website in tutorial overview lists.

Reply:
Done in a25e567